numpy/core/records.py

```diff
@@ -606,6 +629,17 @@ def fromarrays(arrayList, dtype=None, shape=None, formats=None,
     >>> x1[1]=34
     >>> r.a
     array([1, 2, 3, 4])
+    
+    >>> x1 = np.array([1,2,3,4])
+    >>> x2 = np.array(['a','dd','xyz','12'])
+    >>> x3 = np.array([1.1,2,3,4])
```

Comment:
I know the rest of these examples don't follow it, but let's try to use PEP8 here
```suggestion
    >>> x1 = np.array([1, 2, 3, 4])
    >>> x2 = np.array(['a', 'dd', 'xyz', '12'])
    >>> x3 = np.array([1.1, 2, 3,4])
```